assets/js/adminCtrl.js

```diff
@@ -83,8 +83,54 @@ define([
             });
         };
 
+        $scope.getFlairs = function () {
+          var url = "/flair/all";
+          io.socket.get(url, function (data, res) {
+            if (res.statusCode === 200) {
+              $scope.flairs = data;
+              if (data.length === 0) {
+                $scope.flairs[0] = {
+                  name: "",
+                  trades: "",
+                  shinyevents: "",
+                  eggs: "",
+                  sub: ""
+                };
+              }
+            }
+          });
+        };
+
+        $scope.formattedRequirements = function (flair) {
```

Comment:
Ah wait, I get it, it's for just the flair app list. Alright. Maybe we could do this for the app application as well?